.changes/next-release/feature-21a9e26b-edbf-4ad3-a5f4-2662420d4f0d.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "/review: passing referenceTrackerConfiguration to StartCodeFixJob"
```

Comment:
Can we update this? It should describe the change from the end user's perspective